### PR TITLE
Adding github action to auto-increment version in code

### PR DESCRIPTION
**Pull request by maorlx**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (adavidai) [inline .github/workflows/main.yml:1]: Remove comment :)

**Comment by enoodle** [comment]
this will always advance the minor version.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: Update version
+        run: |
+          echo Add other actions to build,
```
Comment (adavidai) [inline .github/workflows/main.yml:26]: Remove 2 first echo commands (or replace with `echo "updating tracer version"`

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          echo test, and deploy your project.
+          git remote add github "https://maorlx:$GITHUB_TOKEN@github.com/$GITHUB_REPOSITORY.git"
```
Comment (adavidai) [inline .github/workflows/main.yml:28]: Let me check which user should be used

**Review by adavidai — COMMENTED** [review]
Some minor comments below

**Review by adavidai — APPROVED** [review]

**Pull request merged**